qa/rpc-tests/wallet_overwintertx.py

```diff
@@ -92,6 +100,22 @@ def run_test (self):
         assert_equal(bci['consensus']['nextblock'], '5ba81b19')
         assert_equal(bci['upgrades']['5ba81b19']['status'], 'pending')
 
+        # Test using expiryheight parameter of createrawtransaction when Overwinter is active in the next block
+        try:
+            self.nodes[0].createrawtransaction([], {}, 0, 499999999)
+        except JSONRPCException,e:
+            assert(False)
```

Comment:
If we aren't expecting a failure, the we shouldn't catch and rethrow, otherwise we won't see the error message in the output (we would just see the assertion failure).

Reply:
The line number of the assertion will be shown in the output so should be ok.  Anyway, I've pushed an update so that the error message will appear.